erpnext/projects/doctype/timesheet/timesheet.html

```diff
@@ -0,0 +1,11 @@
+<div class="stopwatch">
```

Comment:
Looks like this belongs in `timer.js` itself, as a helper function returning the template, and not in `timesheet`.